src/3d/qgsambientocclusionblurentity.cpp

```diff
@@ -0,0 +1,129 @@
+/***************************************************************************
+  qgsambientocclusionblurentity.cpp
```

Comment:
This class looks a lot like `qgsambientocclusionrenderentity.cpp` . Would it make sense to have the common bits on a parent class?